scripts/update-contributions.js

```diff
@@ -17,6 +21,9 @@ Group.findAll({
     'settings'
   ]
 })
+  .tap(groups => {
+    log.verbose('groups', `Found ${groups.length} group(s) to inspect`);
```

Comment:
as mentioned in [other PR](https://github.com/OpenCollective/opencollective-api/pull/404#discussion_r66044623), can use one-liner.